parsl/executors/workqueue/executor.py

```diff
@@ -734,15 +745,58 @@ def submit(self, func, *args, **kwargs):
 
         return fu
 
-    def scale_out(self, *args, **kwargs):
-        """Scale out method. Not implemented.
+    def initialize_scaling(self):
+        """ Compose the launch command and call scale out
+
+        Scales the workers to the appropriate nodes with provider
         """
-        pass
+
+        # Format launch command for the Provider
+        launch_command = self.worker_command.format(hostname=socket.gethostname(), port=self.port)
+        self.worker_command = launch_command
```

Comment:
I think the hostname in `launch_command` here should be configurable? like the `HighThroughputExecutor`

Reply:
Ah, nevermind. I did not realize the launch_command was added by this pr. I will fix it in a further commit then.

Reply:
Hi @btovar, could you take a quick look if the launch_command change you wanted is pushed, please?

Reply:
Done.